app/jobs/reports/base_report.rb

```diff
@@ -47,13 +47,32 @@ def transaction_with_timeout(rails_env = Rails.env)
     end
 
     def save_report(report_name, body, extension:)
+      write_data_to_analytics_logs(report_name, body, extension)
       if !IdentityConfig.store.s3_reports_enabled
         logger.info('Not uploading report to S3, s3_reports_enabled is false')
         return body
       end
       upload_file_to_s3_timestamped_and_latest(report_name, body, extension)
     end
 
+    def write_data_to_analytics_logs(report_name, body, extension)
+      return unless extension == 'json'
+      log_hash = {
+        time: Time.zone.now.iso8601,
+        report_name: report_name,
+        report_body: JSON.parse(body),
```

Comment:
Thanks for making the changes to use a separate log file, I thought we also discussed logging the specific metrics we needed, so that they're easier to query in CW, not logging the entire report bodies?

Reply:
Playing with cloudwatch queries now to see if there's any benefit.  I'm going to mark this as do not merge since i think we need to wait until the devops deploy first is that correct?

Reply:
I mean devops will catch up and read these files when it gets deployed. I think that if we were happy with the shape of the logs, we could deploy this, and understand that it will take a bit before the logs show up in CW

Reply:
Tested out cloudwatch queries.  Parsing the JSON and getting the values into the dashboard is dead simple with the whole report.  i'd prefer getting the entire reports for free like it's coded here.  We would never have to touch idp again and have everything on tap in cloudwatch

Reply:
Spoke to @zachmargolis offline...he convinced me custom entries are the way to go....